R/parallel-taskq.R

```diff
@@ -220,21 +215,22 @@ task_q <- R6::R6Class(
           c(private$tasks$startup[[task_no]], msg$stderr),
           collapse = "\n"
         )
-        abort(
-          paste0(
-            "testthat subprocess failed to start, stderr:\n",
-            msg$error$stderr
+        cli::cli_abort(
+          c(
+            "testthat subprocess failed to start.",
+            " " = "{no_wrap(msg$error$stderr)}"
           ),
           test_file = NULL,
-          parent = msg$error,
```

Comment:
I think this looses the parent error here.

Reply:
I guess now that we collect the stdout/stderr separately, it does not, in most cases.